### PR TITLE
Updating PyMongo to 3.12.0

### PR DESCRIPTION
**Pull request by macintacos**

### CODE REVIEW

File: docsets/PyMongo/docset.json [hunk 1]
```diff
@@ -1,15 +1,19 @@
 {
     "name": "PyMongo",
-    "version": "3.10.1",
+    "version": "3.12.0",
     "archive": "PyMongo.tgz",
```
Comment (Kapeli) [inline docsets/PyMongo/docset.json:4]: You need to update the archive at this path as well.

File: docsets/PyMongo/docset.json [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+      "name": "Julian Torres",
+      "link": "https://github.com/macintacos"
     },
     "aliases": [
         "pymongo"
     ],
     "specific_versions": [
+        {
+            "version": "3.12.0",
+            "archive": "versions/3.12.0/PyMongo.tgz"
```
Comment (Kapeli) [inline docsets/PyMongo/docset.json:15]: This archive doesn't exist. You added `versions/3.12.0/PyMongo.docset.tgz` instead.

**Comment by macintacos** [comment]
@Kapeli ready for another look, sorry about the naming stuff.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**